libsel4/include/interfaces/sel4.xml

```diff
@@ -1194,7 +1194,7 @@
 
     <interface name="seL4_IRQControl" manual_name="IRQ Control" cap_description="An IRQControl capability. This gives you the authority to make this call.">
 
-        <method id="IRQIssueIRQHandler" name="Get" manual_name="Get" manual_label="irq_controlget">
+        <method id="IRQIssueIRQHandler" name="Get IRQ Handler" manual_label="irq_controlget">
```

Comment:
Actually this one needs fixing. It should be

```xml
<method id="IRQIssueIRQHandler" name="Get" manual_name="Get IRQ Handle" manual_label="irq_controlget">
```

(the current change modifies `name`)

Reply:
Seems you fixed it with the rebase.